build/e2e-tests.sh

```diff
@@ -34,6 +34,10 @@ if [ -z "${NODE_IP}" ]; then
     echo "NODE_IP must be set"
     exit 1
 fi
+if [ -z "${SLOW_E2E_THRESHOLD}" ]; then
```

Comment:
Moving purely E2E related variables declarations here (with defaults, from go-in-docker.sh) would make all these ifs a bit more loose for people executing the script directly from build/ without the variable set.

See comment below.